src/main/resources/graphql/stats.graphqls

```diff
@@ -24,6 +24,32 @@ type Stats {
     freeThrowPercentage: Float
 }
 
+input StatsInput {
```

Comment:
As far as I see StatsInput, TeamInput and PlayerInput are not used, I think they should be removed for now

Reply:
Okay, I'm on it! 🏃

Reply:
Done! 🔥